src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -24,6 +24,22 @@
   }
 }
 
+.HeroRecommendation--no-image {
+  justify-content: center;
```

Comment:
I don't think this centered layout is working too well, especially on wide screens. The with-image layout already has a no-image variant for mobile screens so I suggest we use that. This would make the no-image layout similar to the existing one and give us some time to get feedback from UX once it's on dev. Also, I find that the original spacing and top-down, left-aligned flow was well thought out.

This looks good to me in a variety of screens. What do you think?

I also changed `136px` to `132px` so that it's a multiple of 12, which strikes a balance with the other spacing we use.

<details>
<summary>Patch</summary>

```diff
diff --git a/src/amo/components/HeroRecommendation/styles.scss b/src/amo/components/HeroRecommendation/styles.scss
index ee4997f74..f9c959f7f 100644
--- a/src/amo/components/HeroRecommendation/styles.scss
+++ b/src/amo/components/HeroRecommendation/styles.scss
@@ -20,23 +20,19 @@
   }
 
   @include respond-to(extraExtraLarge) {
-    @include margin-start(136px);
+    @include margin-start(132px);
   }
 }
 
 .HeroRecommendation--no-image {
-  justify-content: center;
-
   @include respond-to(extraExtraLarge) {
-    @include padding-start(36px);
+    padding: 36px;
   }
 
   .HeroRecommendation-info {
-    align-items: center;
-    display: flex;
-    flex-direction: column;
-
-    @include margin-start(0);
+    @include respond-to(extraExtraLarge) {
+      margin: 0 132px;
+    }
   }
 }
 

```

</details>

<details>
<summary>Screenshots</summary>

<img width="1360" alt="Screenshot 2019-09-24 11 10 57" src="https://user-images.githubusercontent.com/55398/65530195-17250c80-debd-11e9-92bf-87f5b20ee097.png">
<img width="1129" alt="Screenshot 2019-09-24 11 11 26" src="https://user-images.githubusercontent.com/55398/65530197-17250c80-debd-11e9-831f-f54ac2b41bea.png">
<img width="819" alt="Screenshot 2019-09-24 11 11 51" src="https://user-images.githubusercontent.com/55398/65530199-17250c80-debd-11e9-9e72-77affedd5aac.png">
<img width="477" alt="Screenshot 2019-09-24 11 12 15" src="https://user-images.githubusercontent.com/55398/65530200-17250c80-debd-11e9-8c7c-d40e3c8b7f15.png">


</details>

Reply:
I personally agree it looks better, but it doesn't match the mock in the issue. The issue description even explicitly calls out "It should change the layout to centred...", so I'm not sure we should disregard both of those things. I suppose we can land what you've suggested and get feedback from UX, but don't we generally try to match the mock as closely as possible before landing?

Reply:
oh, sorry! When I asked "do we have a Sketch mock" and you said no I thought you were implying that we didn't have any mock at all, my bad 🤦‍♂ 

Let me take another look.

Reply:
Sorry, I misinterpreted the question. I thought you were looking for a Sketch mock so we could confirm exact sizes and spacing. We don't have a Sketch file, but there is the image included in the original issue at https://github.com/mozilla/addons-frontend/issues/8629#issue-495881771.

I think we're on the same page now. 😄